backend-rust/src/indexer.rs

```diff
@@ -617,6 +639,7 @@ struct BlockData {
     chain_parameters:     ChainParameters,
     tokenomics_info:      RewardsOverview,
     total_staked_capital: Amount,
+    special_events:       Vec<SpecialTransactionOutcome>,
```

Comment:
We could make this a Stream (As it is provided by the SDK.).

Since it diverges with the rest of the properties on this struct, I would probably leave it as is.